src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -801,7 +801,7 @@ def create(notebook_path: str | Path | None = None, **kwargs):
             ]
 
         # add RemoveAfter tag for test job cleanup
-        date_to_remove = (datetime.now() + JOBS_PURGE_TIMEOUT).strftime("%Y-%m-%d")
+        date_to_remove = (datetime.utcnow() + JOBS_PURGE_TIMEOUT).isoformat()
```

Comment:
```suggestion
        date_to_remove = (datetime.utcnow() + JOBS_PURGE_TIMEOUT).strftime("%Y%m%d%H")
```
roll-up to hour. also extract this to a function.